src/lib/markdown/utils.ts

```diff
@@ -0,0 +1,112 @@
+// 埋め込み?
+// * table
+// * 取り消し線
+//
+// 付けたい機能
+// * footnote
+// * highrightjs
+// * katex
+// * container
+// * front-matter
+//
+// 要件に応じて
+// * anchor
+// * toc-done-right
+// * ins
+// * mark
+// * emoji
+
+import MarkdownIt from 'markdown-it'
+import MarkdownItFootnote from 'markdown-it-footnote'
+import MarkdownItFrontMatter from 'markdown-it-front-matter'
+import MarkdownItHighlightjs from 'markdown-it-highlightjs'
+import MarkdownItKatex from 'markdown-it-katex'
+import MarkdownItContainer from 'markdown-it-container'
+import MarkdownItPlainText from 'markdown-it-plain-text'
+
+type Status = 'error' | 'success' | 'warning' | 'info'
+
+// const statusColor: { [key in Status]: string } = {
+//   error: 'red',
+//   success: 'green',
+//   warning: 'yello',
+//   info: 'blue',
+// }
+
+const statusDiv: { [key in Status]: string } = {
+  // TODO: Emoji は Icon に変法
+  error:
+    "<div class='bg-red-50 border border-red-200 rounded-md text-sm text-red-700 m-2 px-2 flex' role='alert'><div class='flex-shrink-0 self-center text-xl'>❌</div><div class='ml-4'>",
+  success:
+    "<div class='bg-green-50 border border-green-200 rounded-md text-sm text-green-700 m-2 px-2 flex' role='alert'><div class='flex-shrink-0 self-center text-xl'>✅</div><div class='ml-4'>",
+  warning:
+    "<div class='bg-yellow-50 border border-yellow-200 rounded-md text-sm text-yellow-700 m-2 px-2 flex' role='alert'><div class='flex-shrink-0 self-center text-xl'>⚠️</div><div class='ml-4'>",
+  info: "<div class='bg-blue-50 border border-blue-200 rounded-md text-sm text-blue-700 m-2 px-2 flex' role='alert'><div class='flex-shrink-0 self-center text-xl'>ℹ️</div><div class='ml-4'>",
+}
+
+const statusTitle: { [key in Status]: string } = {
+  error: "<p class='font-bold text-red-800'>",
+  success: "<p class='font-bold text-green-800'>",
+  warning: "<p class='font-bold text-yellow-800'>",
+  info: "<p class='font-bold text-blue-800'>",
+}
+
+const containerRegExp = (status: Status) => {
+  const re = new RegExp(`^${status}+(.*)$`)
+  return re
+}
```

Comment:
```suggestion
const containerRegExp = (status: Status): RegExp =>  new RegExp(`^${status}+(.*)$`)
```

わざわざreturnしなくてもいいかも！